app/assets/src/components/common/metadata_manual_input.scss

```diff
@@ -47,7 +47,7 @@
 
     .input {
       margin: $space-xxs $space-m $space-xxs 0;
-      width: 130px;
+      width: 110px; // see also getColumnWidth
 
       &.extraWidth {
```

Comment:
There's actually a bug with these values (and the bug existed prior to your change). See screenshot. This occurs when you add a bunch of columns to the MetadataManualInput.

![Screen Shot 2019-12-18 at 12 26 33 PM](https://user-images.githubusercontent.com/837004/71120732-de31c680-2191-11ea-87bb-bde0ae823365.png)

To prevent this, the width of the input needs to be smaller than the width of the column. Perhaps by about 15px. So if the column is 110, the width of the input should be 95px.

There's a way to export variables from `.scss` files, which you can use to share the constants between scss and javascript. See the `:export` block in `pipeline_viz.scss`.  By sharing a constant for the "buffer" (e.g. 15px), this can help prevent similar bugs in the future.